pkg/registry/core/pod/etcd/eviction.go

```diff
@@ -104,14 +105,18 @@ func (r *EvictionREST) Create(ctx api.Context, obj runtime.Object) (runtime.Obje
 	return &unversioned.Status{Status: unversioned.StatusSuccess}, nil
 }
 
-func (r *EvictionREST) checkAndDecrement(namespace string, pdb policy.PodDisruptionBudget) (ok bool, err error) {
+func (r *EvictionREST) checkAndDecrement(namespace string, podName string, pdb policy.PodDisruptionBudget) (ok bool, err error) {
 	if pdb.Status.PodDisruptionsAllowed < 0 {
 		return false, fmt.Errorf("pdb disruptions allowed is negative")
 	}
 	if pdb.Status.PodDisruptionsAllowed == 0 {
 		return false, nil
 	}
 	pdb.Status.PodDisruptionsAllowed--
+	if pdb.Status.DisruptedPods == nil {
+		pdb.Status.DisruptedPods = make(map[string]unversioned.Time)
+	}
+	pdb.Status.DisruptedPods[podName] = unversioned.Time{Time: time.Now()}
```

Comment:
Can you add a comment for why we are capturing the time here? Will be useful for future reviewers.